lib-database/src/commonMain/kotlin/com/ustadmobile/core/db/dao/ContentEntryDao.kt

```diff
@@ -161,6 +109,19 @@ abstract class ContentEntryDao : BaseDao<ContentEntry> {
     @JsName("findByTitle")
     abstract fun findByTitle(title: String): DoorLiveData<ContentEntry?>
 
+    @Query("""
+       SELECT COALESCE((SELECT cjIsMeteredAllowed 
```

Comment:
This subquery needs a LIMIT 1. In reality, there never really should be more than one job for a given content entry, but better to have it just in case.